app/scripts/contentscript.js

```diff
@@ -52,4 +52,22 @@ const start = () => {
   }
 };
 
+(function () {
+  let once = false;
+  // TODO(34913): "document" because LavaMoat blocks props
+  // TODO(34913): definitions against the globalThis (window)
```

Comment:
reposting my comment on this from elsewhere:
That's not true. document is shared between the page realm and the contentscript while window is not. Defining the property on window will not work because of how the contentscript is icolated from the page.
Chrome and FF have different approaches to the isolation. Both somewhat cursed.
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Content_scripts#content_script_environment